### PR TITLE
Minor grammatical changes

### PR DESCRIPTION
**Pull request by janschill**
Just a few corrections I had noticed reading the documentation.

- [x] Any changes relevant to users are recorded in the `CHANGELOG.md`.
- [x] The documentation has been updated, if necessary.
- [x] New code is annotated.
- [x] Changes are covered by tests.
---

### CODE REVIEW

**Comment by schlenk** [comment]
Thanks for contributing. Looks good. 

Seems our CI is currently broken due to cryptography adding rust to their toolchain. But thats unrelated to the change.

File: doc/examples/rp.rst [hunk 1]
```diff
@@ -48,10 +48,10 @@ you will learn what information to look for.
 Issuer discovery
 ----------------
 
-OIDC uses `webfinger`_ to to do the OP discovery. In very general terms this means
+OIDC uses `webfinger`_ to do the OP discovery. In very general terms this means
 that the user that accesses the RP provides an identifier. There are a number
 of different syntaxes that this identifier can adhere to. The most common is
-probably the e-mail address syntax. It's something that looks like an e-mail
+probably the email address syntax. It's looks like an email
```
Comment (schlenk) [inline doc/examples/rp.rst:54]: "It looks like an email" not "It's"

**Review by schlenk — CHANGES_REQUESTED** [review]
Looks mostly good.

File: doc/examples/rp.rst (hunk 1 above)
Comment (janschill) [reply doc/examples/rp.rst:54]: Ah, good catch. I will quickly change it.

Comment (janschill) [reply doc/examples/rp.rst]: I pushed a change.

**Review by schlenk — APPROVED** [review]
LGTM

**Comment by schlenk** [comment]
I'll merge it once CI is back on track.

**Pull request merged**